eng/targets/Java.Common.targets

```diff
@@ -0,0 +1,87 @@
+<Project DefaultTargets="Build">
+
+  <PropertyGroup>
+    <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)' == ''">$(MSBuildProjectDirectory)\obj\</BaseIntermediateOutputPath>
+    <IntermediateOutputPath>$([MSBuild]::NormalizeDirectory('$(BaseIntermediateOutputPath)'))$(Configuration)\</IntermediateOutputPath>
+    <BuildDependsOn>
+      PrepareForBuild;
+      ResolveProjectReferences;
+      _Build;
+    </BuildDependsOn>
+    <JavaBuildArgs Condition="'$(JavaBuildArgs)' == ''">../gradlew $(GradleOptions) compileJava</JavaBuildArgs>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <JavaFiles Include="src\**\*.java" />
+    <JavaFiles Include="*.javaproj" />
+    <JavaFiles Include="build.gradle" />
+
+    <BuildOutputFiles Include="$(BaseIntermediateOutputPath)build-sentinel" />
+  </ItemGroup>
+
+  <Target Name="Restore" />
+
+  <PropertyGroup>
+    <PackDependsOn Condition=" '$(NoBuild)' != 'true' ">
+      $(PackDependsOn);
+      Build
+    </PackDependsOn>
```

Comment:
This setting is repeated below. Remove one of them. Also, is the order correct here i.e. should `Build` be first❔